Score Keeper of NottinghamTests-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
```

Comment:
Unclear why Xcode insisted this was necessary to run tests, but here we have it.